Standards/scs-01XX-v1-secure-connections.md

```diff
@@ -169,6 +169,12 @@ This option would still offer improvements over arbitrary OpenStack clouds by es
 
 ## Open questions
 
+### Choosing the best protection for the libvirt hypervisor interface
+
+As described in the design considerations section, there are multiple ways of securing the libvirt interface using TLS or SSH.
+Each approach holds its own challenges and requires a robust provisioning and lifecycle mechanism for the cryptographic assets (e.g. client keys or certificates) to ensure proper configuration of all involved nodes, even if the set of nodes changes.
+Aside from extensive testing required to select the best approach, this goes beyond simple component configuration and relies on sophisticated key management which this standard alone does not provide.
```

Comment:
Should we then maybe reference the Key-Manager standard here, even if it's still not merged?

Reply:
The Key Manager standard is currently limited in scope, specifically to the OpenStack Key Manager API (Barbican).
Barbican is not a PKI solution for X.509 or anything remotely close. It is much simpler - just a secret store basically.
We would need key rotation, client certificate issuance, CA functionality for the TLS approach or SSH identity management for the SSH approach. Barbican offers none of this as far as I'm aware.

Reply:
Ack.